modules/nextflow/src/main/groovy/nextflow/container/PodmanBuilder.groovy

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright 2013-2019, Centre for Genomic Regulation (CRG)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package nextflow.container
+import groovy.transform.CompileStatic
+/**
+ * Helper methods to handle podman containers
+ *
+ * @author Paolo Di Tommaso <paolo.ditommaso@gmail.com>
+ * @author tbugfinder <github@online.ms>
+ */
+@CompileStatic
+class PodmanBuilder extends ContainerBuilder<PodmanBuilder> {
+
+    private boolean sudo
+
+    private boolean remove = true
+
+    private boolean userEmulation
+
+    private String registry
+
+    private String name
+
+    private boolean tty
+
+    private static final String USER_AND_HOME_EMULATION = '-u $(id -u) -e "HOME=${HOME}" -v /etc/passwd:/etc/passwd:ro -v /etc/shadow:/etc/shadow:ro -v /etc/group:/etc/group:ro -v $HOME:$HOME'
+
+    private String removeCommand
+
+    private String killCommand
+
+    private kill = true
+
+    private boolean legacy
```

Comment:
This flag is not needed.

Reply:
Do you mean `kill` or about `legacy`?

Reply:
removed legacy flag with
commit 18eca201f4a77b8a9045f84a88fe13d7ae56be32

Reply:
Sorry, I mean `legacy`. This was related to old docker options.